hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestAbfsRestOperationMockFailures.java

```diff
@@ -188,7 +187,7 @@ private void testClientRequestIdForStatusRetry(int status,
     int[] statusCount = new int[1];
     statusCount[0] = 0;
     Mockito.doAnswer(answer -> {
-      if (statusCount[0] <= 5) {
+      if (statusCount[0] <= 10) {
```

Comment:
Since we are updating the metrics based on status code, the getStatusCode() method is called more than the above mentioned number. I think we should not have code where we specifically mention number of times a method is called as logic to return something as that can change frequently.